### PR TITLE
Milestone1

### PR DESCRIPTION
**Pull request by Adewunmi97**
- Implemented only a basic UI with plain HTML
- Created a collection that keeps a list of books (hint: you can use an array of objects for that).
- Created a function to add a new book to the collection, with title and author.
- Created a function to remove a book from the collection (hint: you can use the array filter() method).
- Displayed all books saved in the collection in the top part of the page.
- When a user clicks the "Add" button:
- A new book is added to the collection.
- The new book is displayed on the page.
- When a user clicks the "Remove" button:
- The correct book is removed from the collection.
- The correct book disappears from the page.
- Data is preserved in the browser's memory by using localStorage.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  <!-- <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    /> -->
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body onload="startTime()">
```
Comment (henatan99) [inline index.html:17]: - [ ] This `startTime()` function is throwing console error on the browser. Kindly fix it! :pray: 

Screenshot from 2021-09-13 22-12-10

File: main.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+const bookTitle = document.querySelector('.title');
+const bookAuthor = document.querySelector('.author');
+const bookList = document.querySelector('.book-list');
+class Storage {
+  constructor() {
+    this.collection = [];
+  }
+
+  static addCollection(newBook) {
+    this.collection.push(newBook);
```
Comment (henatan99) [inline main.js:11]: - [ ] OPTIONAL: Array `concat` method is preferred over `push` method for the sake of data integrity as it does not alter the original array. :+1:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @Adewunmi97 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [x] A good descriptive readme is an important part of a preofessional repo. Kindly use the Microverse readme [template](https://github.com/microverseinc/readme-template) and add minimal description of your project. :pray: 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — APPROVED** [review]
### **Hi @Adewunmi97 && @chuanqizhu0820,** 

### **Project Approved ✅**

Approved


### **Optional**

- Please remove the Onload=startTime() from your HTML body it causing a problem.

kskska



Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**